### ISSUE TITLE
UI designing 

### ISSUE THREAD

**Issue by niralverma** [praveenscience/devternship.lgmsoc.co#1]
Create pages, decide a theme color, layout, typography, and other design elements.

**Comment by RiyaGupta89** [comment]
Can I have a some demo UI that how the pages should look alike. That would be really helpful, also I would like to work on this issue.

**Comment by niralverma** [comment]
> Can I have a some demo UI that how the pages should look alike. That would be really helpful, also I would like to work on this issue.

Yes sure. I will send the basic demo soon. Then you can modify it.

**Comment by Akshu-on-github** [comment]
Few ideas that I had based on this [LGM SoC prototype](https://rohan-kulkarni-25.github.io/LGM-SOC/pages/participantsguidelines.html). It's a work in progress, let me what you think of this [Figma Design](https://www.figma.com/file/kvHJVCsqtQvLGLQtC7DVma/LGMSoc?node-id=0%3A1)?

### Features:
- [ ] Tentative colour scheme for Light and Dark modes (suggested: use alternatively)
  - Light mode colours:
    - Emphasis: #001585
    - Right Background: #FFFFFF
    - Left Background: #001585 to #000830 Linear Gradient
    - Text: #000830
  - Dark mode colours:
    - Emphasis: #9FBEFF
    - Text: #FFFFFF
    - Background: #001585 to #000830 Linear Gradient
- [ ] Design for basic components:
    - Navbar
    - Questions
    - Answers
    - Navigation buttons
- [ ] Free, copyright-free images from Unsplash
- [ ] Graphic blob (aesthetic)

**Comment by praveenscience** [comment]
@Akshu-on-github This is extremely good! 🔥
